agenta-backend/agenta_backend/utills/common.py

```diff
@@ -0,0 +1,8 @@
+import string
+import secrets
+
+
+def generate_invitation_token(token_length: int = 16):
+    characters = string.ascii_letters + string.digits
+    token = "".join(secrets.choice(characters) for _ in range(token_length))
+    return token
```

Comment:
This is good. 

However, I would recommend you utilize Python's built-in cryptographic module (secrets), which is designed for generating secure tokens and is generally considered a good practice for generating tokens for authentication, invitations, or similar purposes.  


```suggestion
def generate_invitation_token(token_length: int = 16):
    token = secrets.token_urlsafe(token_length)
    return token
```